### ISSUE TITLE
Non-transformed JSON is doubled

### ISSUE THREAD

**Issue by stevage** [perliedman/reproject#37]
```
echo '{"foo":"hi"}' | reproject
{
  "foo": "hi"
}
{
  "foo": "hi"
}
```

I don't know what the expected behaviour when receiving non-GeoJSON is, but probably not this. :) 

(This behaviour seems to have always been like this)

**Comment by stevage** [comment]
Yes, still happens with 1.2.1.

**Comment by perliedman** [comment]
1.2.1 doesn't have the fix for #34, though - haven't gotten around to actually making a release.

**Comment by perliedman** [comment]
Published 1.2.2 just now, and it looks like the example you posted above works properly with that, so I'm closing this for now!

**Comment by stevage** [comment]
I'm still seeing the problem as at commit 2c1e862227d59
